src/client/components/Navigation/NavGeo/Layer/ToggleControl/ToggleControl.scss

```diff
@@ -30,6 +30,16 @@
       color: white;
     }
   }
+
+  &.legend {
```

Comment:
`div.legend` didn't work here. An alternative would be `&.data-cell.legend`, or simply `&.legend` as it is now.